mentoring/programs/mentoring-lead.md

```diff
@@ -0,0 +1,40 @@
+# Mentoring Lead Handbook
```

Comment:
This should not be the mentoring lead; this should be the 3rd Party Mentoring Lead. 

Mentoring lead would assume all programs including group mentoring, new contributor workshop, and more. That is too much for one person